applovin_max/example/pubspec.yaml

```diff
@@ -30,3 +30,5 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+  assets:
+    - assets/applovin_logo.png
```

Comment:
Curious, but =what would be the consequences of omitting this?

Reply:
Omitting results in an exception: 'Asset not found.'  But, I realized this needs only the directory. 
I tested it without `/`.

```
  assets:
    - assets/
```